NuGet.config

```diff
@@ -16,7 +16,10 @@
         <add key="repositoryPath" value="..\packages" />
     </config>
   <packageSources>
+    <add key="LocallyBuiltApplicationInsightsDebugPackages" value="..\bin\Debug\NuGet" />
```

Comment:
Unfortunately it is intentional, we hope to be able to remove it with the new project format soon.